code/stable/ssp/SRS/HTML/SSP_SRS.html

```diff
@@ -3071,7 +3071,7 @@ <h3>Data Definitions</h3>
                     <tr>
                       <th>Equation</th>
                       <td>
-                        \[\symbf{h}=\frac{1}{2} \left({\symbf{h}^{\text{R}}}+{\symbf{h}^{\text{L}}}\right)\]
+                        \[\symbf{h}=\frac{1}{2} \left({{\symbf{h}^{\text{R}}}}_{i}+{{\symbf{h}^{\text{L}}}}_{i}\right)\]
```

Comment:
This makes sense to me. However, it looks like the LHS of the `=` should be `h_i` too. Since we use a `QDefinition` to define `h`, it might not be quite fit well to say "forall i. h_i equals to...". For this type error, it might be best that we add vector scaling and vector addition.

I've got a bunch of code work to do today regarding cleaning up some typing rules today, but I'm not sure if I'll be able to get to this. I'll open a ticket as blocking this.